frontend/src/components/TruncatedText.vue

```diff
@@ -92,7 +92,7 @@ div.popover-header {
 }
 .truncated-text {
    font-size: 1em;
-   font-weight: bold;
```

Comment:
This component is used all over the UI. Making all bold is probably not the best. Change this to a config property instead